### ISSUE TITLE
Changing the header and body fonts?

### ISSUE THREAD

**Issue by dietsdigested** [matcornic/hugo-theme-learn#93]
Hey, I'm really new to Hugo (and making websites in general), but I love Grav's Learn theme and I'm so happy you made this Hugo version.

However, I'm not a huge fan of the fonts, and I wanted to use my own font combos off of Google fonts. I found a website that gives a tutorial to override the fonts, but I was wondering if there was a particular place that this was written into the Learn theme? If so, how do I go about just changing it?

I'm also wondering if I need to put the font files into the local folders in Hugo.

Thank you!

**Comment by matcornic** [comment]
Hi @dietsdigested 

I think you can do it by overriding CSS rules about fonts by creating your own `custom-header.html`  like explained [here](https://learn.netlify.com/en/basics/style-customization/)

https://github.com/matcornic/hugo-theme-learn/blob/6562adeb7b3ead1e199272b450a61ad07faf48f4/static/css/theme.css#L18-L36

**Comment by dietsdigested** [comment]
Ah, so it's the custom-header.html that I need to make an overwrite for. I wasn't sure which one had the fonts; I had found them in theme.css like you said, but wasn't sure if the custom-header just did stuff for the header on the page because I barely know what I'm doing. Thank you so much for your help, I needed pretty much exactly what you posted for me.

**Comment by dietsdigested** [comment]
Okay @matcornic so I added a custom-header.html file and put this in there:
`</script>
</style>
<style type="css">
@font-face {
   font-family: 'Eczar', serif;
   src: url("..public\fonts\Eczar-Regular.ttf");
   font-style: normal;
   font-weight: 200;
}
@font-face {
   font-family: 'Gentium Basic', serif;
   font-style: normal;
   font-weight: 300;
   src: url("..public\fonts\GenBasR.ttf") format("embedded-opentype")
}
</style>
</head>`

(right above that is my Google ads html, all of this is within the <head> so I hope that's correct)

I also downloaded the fonts and added the .ttf files to the public/fonts folder, and it won't change the fonts on the localhost site when I run a test server. My problem is that I really don't know where all the font callbacks are located within the files, and I'm assuming that it is pulling them from the theme's folders, I just can't figure out where it is.

Thanks again for your help, I truly appreciate this.

**Comment by matcornic** [comment]
`custom-header`is just called on this line https://github.com/matcornic/hugo-theme-learn/blob/master/layouts/partials/header.html#L36. Meaning you don't have to add another `</head>` in the file.

To understand how Hugo handles files structure, you need to follow Hugo's documentation. [Getting started](https://gohugo.io/getting-started/usage/) and [directory structure](https://gohugo.io/getting-started/directory-structure/) are a good start.

That said, the `public` folder is the one that is generated by Hugo, you **must not** add/remove/modify files in this folder as it's generated. Instead, you need to add your files in your `static` folder. Files in this folder will automatically available at the root of your website like `/fonts/something.ttf` when you have the structure `/static/fonts/something.ttf` . Same thing happens in the theme itself : https://github.com/matcornic/hugo-theme-learn/tree/master/static/fonts. And every file from the theme can be overridden like explained [here](https://gohugo.io/themes/customizing/#override-static-files)

**Comment by dietsdigested** [comment]
Woohoo! I got it to work, thanks to you explaining the `static` folder, I was able to put the fonts there and make a copy of the `theme.css`, and tinker with the fonts from there. Thanks again for your help, but I have one more question:

how do I change in the `theme.css` file which font is going to the sidebar? Right now it's using the body font for the sidebar, and I actually want the h1 font to go to the sidebar. Can't seem to find the specific part that points out which font the sidebar inherits, though I can find certain aspects of it like font size and line spacing.
